nixpkgs/github.json

```diff
@@ -1,6 +1,6 @@
 {
   "owner": "nixos",
-  "repo": "nixpkgs-channels",
-  "rev": "754763ff4ba1dd03fe3fad3a0fea36d2e39f5860",
-  "sha256": "10752kda1rzljlpcchi826hmbc8853vnbg9rkh7s89mxq6yjnm15"
+  "repo": "nixpkgs",
```

Comment:
We should keep this as nixpkgs-channels so we can get branch updates once ob thunk happens.

Reply:
And bonus, it would write down the branch name.